### PR TITLE
update with tuned profile

### PR DESCRIPTION
**Pull request by tfevriermsft**
first attempt to propose the tuned profile we have used for our OLTP and DW benchmarks and reviewed with Red Hat.

### CODE REVIEW

**Comment by PRMerger7** [comment]
@tfevriermsft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@VanMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by VanMSFT** [comment]
In review

**Comment by PRMerger15** [comment]
@tejasaks, you can only sign off in the main PR conversation, not in review comments.

**Comment by PRMerger18** [comment]
@VanMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/linux/sql-server-linux-performance-best-practices.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ These are the recommended Linux Operating System settings related to high perfor
 
 
 > [!Note]
-> For Red Hat Enterprise Linux (RHEL) users, the throughput-performance profile will configure these settings automatically (except for C-States).
+> For Red Hat Enterprise Linux (RHEL) users, the tuned throughput-performance profile configure these settings automatically (except for C-States). Starting with RHEL 8.0, a /usr/lib/tuned built-in mssql profile was co-developed with Red Hat and offers finer Linux performance related tunings for SQL Server workloads. This profile includes the RHEL throughput-performance profile and we present its definitions below for your review with other Linux distros and RHEL releases without this profile.
```
[the remark this answers is not in the document]
Comment (VanMSFT) [reply docs/linux/sql-server-linux-performance-best-practices.md]: Thanks Tejas! I added the link

**Comment by VanMSFT** [comment]
#sign-off

**Comment by PRMerger18** [comment]
@VanMSFT: I'm sorry - only the author of this article, @tejasaks, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by tejasaks** [comment]
#sign-off

**Comment by ktoliver** [comment]
Thanks for reviewing the PR; merging. 

@VanMSFT It's preferred to update the metadata `author` and `ms.author` attributes in the private repo, for future reference. Thanks!

**Comment by VanMSFT** [comment]
@ktoliver - Thanks, Kristine!

**Comment by tfevriermsft** [comment]
Thanks Kristine, Van

I would like to know if i have to do anything now. I can see that the changes were merged but the web site is still reporting the prior content. Does it mean its refresh will be done later?
Or do i need to push official the changes?

Please let me know.
Thank you again.

Once these updates are uploaded, i will add new updates.

Best regards
Thierry Fevrier
Microsoft R&D Data-SQL Server
"Satisfaction lies in the effort, not the attainment. Full effort is full victory" - Mohandas Karamchand Gandhi.

**Comment by ktoliver** [comment]
Hi @tfevriermsft - I'm not involved in publishing, and I'm not familiar with the publishing process for this public repo (and I don't know where it might be documented). 

@rothja Do you know when an update in a merged PR might appear on the live site?

**Comment by tfevriermsft** [comment]
It takes about a day from what I have seen to go from the branch we work upon to live branch and then publishing, which seems to happen twice (morning and late afternoon PST). It's live now.

~Tejas Shah | Principal Program Manager | Data Platform - SQL Server

From: Thierry Fevrier <tfevrier@microsoft.com>
Sent: Saturday, February 22, 2020 4:01 AM
To: MicrosoftDocs/sql-docs <reply@reply.github.com>; MicrosoftDocs/sql-docs <sql-docs@noreply.github.com>
Cc: Mention <mention@noreply.github.com>; Van To (SQL) <vanto@microsoft.com>; Tejas Shah <tejas.a.shah@microsoft.com>
Subject: RE: [MicrosoftDocs/sql-docs] update with tuned profile (#4191)

Thanks Kristine, Van

I would like to know if i have to do anything now. I can see that the changes were merged but the web site is still reporting the prior content. Does it mean its refresh will be done later?
Or do i need to push official the changes?

Please let me know.
Thank you again.

Once these updates are uploaded, i will add new updates.

Best regards
Thierry Fevrier
Microsoft R&D Data-SQL Server
"Satisfaction lies in the effort, not the attainment. Full effort is full victory" - Mohandas Karamchand Gandhi.

**Comment by tfevriermsft** [comment]
Thanks Tejas,

I will add some updates for your review in the coming days. One of them is the lazytime is the server reset validations under SQL workloads are successful.

Best regards
Thierry Fevrier
Microsoft R&D Data-SQL Server
"Satisfaction lies in the effort, not the attainment. Full effort is full victory" - Mohandas Karamchand Gandhi.

From: Tejas Shah <tejas.a.shah@microsoft.com>
Sent: Saturday, February 22, 2020 1:12 AM
To: Thierry Fevrier <tfevrier@microsoft.com>; MicrosoftDocs/sql-docs <reply@reply.github.com>; MicrosoftDocs/sql-docs <sql-docs@noreply.github.com>
Cc: Mention <mention@noreply.github.com>; Van To (SQL) <vanto@microsoft.com>
Subject: RE: [MicrosoftDocs/sql-docs] update with tuned profile (#4191)

It takes about a day from what I have seen to go from the branch we work upon to live branch and then publishing, which seems to happen twice (morning and late afternoon PST). It's live now.

~Tejas Shah | Principal Program Manager | Data Platform - SQL Server

From: Thierry Fevrier <tfevrier@microsoft.com<mailto:tfevrier@microsoft.com>>
Sent: Saturday, February 22, 2020 4:01 AM
To: MicrosoftDocs/sql-docs <reply@reply.github.com<mailto:reply@reply.github.com>>; MicrosoftDocs/sql-docs <sql-docs@noreply.github.com<mailto:sql-docs@noreply.github.com>>
Cc: Mention <mention@noreply.github.com<mailto:mention@noreply.github.com>>; Van To (SQL) <vanto@microsoft.com<mailto:vanto@microsoft.com>>; Tejas Shah <tejas.a.shah@microsoft.com<mailto:tejas.a.shah@microsoft.com>>
Subject: RE: [MicrosoftDocs/sql-docs] update with tuned profile (#4191)

Thanks Kristine, Van

I would like to know if i have to do anything now. I can see that the changes were merged but the web site is still reporting the prior content. Does it mean its refresh will be done later?
Or do i need to push official the changes?

Please let me know.
Thank you again.

Once these updates are uploaded, i will add new updates.

Best regards
Thierry Fevrier
Microsoft R&D Data-SQL Server
"Satisfaction lies in the effort, not the attainment. Full effort is full victory" - Mohandas Karamchand Gandhi.

**Pull request merged**